quantum/quantum.c

```diff
@@ -215,6 +215,9 @@ bool pre_process_record_quantum(keyrecord_t *record) {
 /* Get keycode, and then call keyboard function */
 void post_process_record_quantum(keyrecord_t *record) {
     uint16_t keycode = get_record_keycode(record, false);
+    #if defined(POINTING_DEVICE_ENABLE) && defined(POINTING_DEVICE_AUTO_MOUSE_ENABLE)
+        process_auto_mouse(keycode, record);
+    #endif
```

Comment:
Recommend changing the function to a bool and moving this to `process_record_quantum` above `process_record_kb` (see latest updates in my PR) as there was a pretty serious issue with keyrecords not going through `process_auto_mouse` if at any point in the process_record stack they returned false (which often is the case for custom keys or overrides)

Reply:
O, thanks! I am going to reddo this right at one point this week and cherry pick your commits instead of copying the code manually (my git-fu is rusty). That way I can keep up with your most recent updates and I won't mess with the commit history when Sadek merges it. Unless Sadek gets around to it before me!

Reply:
yeah that sounds like a good plan I think I have commit all the biggest changes now and the code should be pretty stable from here on out (hopefully....)